modules/tinymce/src/core/test/ts/browser/EditorUploadTest.ts

```diff
@@ -18,6 +19,7 @@ UnitTest.asynctest('browser.tinymce.core.EditorUploadTest', (success, failure) =
   Theme();
 
   let testBlobDataUri;
+  let dataImgFilter: (img: HTMLImageElement) => boolean;
```

Comment:
A note, more than anything: when we write up all the caveats for when we revisit this, should make sure we note why this was necessary. And probably include a "please rewrite this as not a LegacyUnit test".